méli-mélo/th-empathy/death.html

```diff
@@ -0,0 +1,123 @@
+---
+title: Life Journey Death home page layout
+dateModified: 2025-02-06
+description: Homepage CSS styles
+lang: en
+pageclass: cnt-wdth-lmtd
+css:
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
+- style.css
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.css
+script:
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.js
+- empathy.js
+layout: without-h1
+---
+<h1 property="name" id="wb-cont" class="empathy">{{ page.title }}</h1>
+<div class="mrgn-tp-lg">
+	<div class="row wb-eqht-grd gc-empathy provisional wb-init">
+		<div class="col-xs-12 col-md-8 empathy services-and-information small">
+			<div class="row wb-eqht-grd">
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<h4><a href="#" class="stretched-link">Preparing for end of life</a></h4>
```

Comment:
I believe these should be `<h2>`. Please also update the code sample.
```suggestion
					<h2><a href="#" class="stretched-link">Preparing for end of life</a></h2>
```

Reply:
Resolved in latest commit